andela/mmdp_site
# MMDP-694 Add focus area target to stakeholders' endpoint

**What does this PR do?**

Adds target to stakeholders' endpoint so as to calculate target completion percentage

**Description of Task to be completed?**

Add focus area target to stakeholders' endpoint so as to calculate target completion percentage

**How should this be manually tested?**

- `git fetch`
- `git checkout ft-impact-factor-MMDP-694`
- `yarn dev`
-  use postman to hit `http://0.0.0.0:3000/api/v1/location?state=Edo&focusAreaName` 
- notice that for each stakeholder, under the beneficiaries dictionary, the focus area targets are now returned with the rest of the data

**Any background context you want to provide?**

- this is just to facilitate the calculation of the target % completion needed for the impact factor table on the website

**What are the relevant Jira issues?**

[MMDP-694](https://viisaus.atlassian.net/browse/MMDP-694)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/40663828/61785288-10720e80-ae14-11e9-81c2-10b265bd544b.png)


**Questions:**
N/A
